lib/src/SofaGLFW/SofaGLFWBaseGUI.cpp

```diff
@@ -113,8 +114,8 @@ sofa::component::visualmodel::BaseCamera::SPtr SofaGLFWBaseGUI::findCamera(sofa:
 
 bool SofaGLFWBaseGUI::createWindow(int width, int height, const char* title, bool fullscreenAtStartup)
 {
-    //glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, 2);
-    //glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 0);
+    imgui::imguiInit();
```

Comment:
I suppose this will make a compilation error if imgui is not enabled ?

Reply:
No. In the current proposition, there is a #ifdef around the content of the function. In a next iteration, we could put it around the function call instead.